apps/web/src/app/[lang]/dho/[id]/layout.tsx

```diff
@@ -40,7 +40,7 @@ export default async function DhoLayout({
     getSpaceDetails({ spaceId: BigInt(spaceFromDb.web3SpaceId as number) }),
   );
   return (
-    <div className="flex max-w-[--spacing-container-2xl] mx-auto">
+    <div className="max-w-[--spacing-container-2xl] mx-auto">
       <Container className="flex-grow min-w-0">
```

Comment:
Should be possible to do this (maybe after rebase on main).
```suggestion
    <div className="max-w-container-2xl mx-auto">
      <Container className="flex-grow min-w-0">
```